@here/olp-sdk-dataservice-read/lib/utils/RequestBuilderFactory.ts

```diff
@@ -21,4 +21,170 @@
  * @deprecated This file will be removed by 02.2021. Please use the same from  the `@here/olp-sdk-core` package.
  */
 
-export * from "@here/olp-sdk-core/lib/utils/RequestBuilderFactory";
+import {
+    ApiCacheRepository,
+    ApiName,
+    DataStoreRequestBuilder,
+    getEnvLookUpUrl,
+    HRN,
+    HttpError,
+    OlpClientSettings
+} from "@here/olp-sdk-core";
+import { LookupApi } from "@here/olp-sdk-dataservice-api";
+const MILLISECONDS_IN_SECOND = 1000;
+
+/**
+ * @deprecated This class will be removed by 02.2021. Please use the same from  the `@here/olp-sdk-core` package.
+ * A helper utils that makes the `Request` object with the base URLs of the API Lookup Service, token callback, and download manager.
+ *
+ * Also, you can use it to get the base URLs of the API Lookup Service.
+ */
+export class RequestFactory {
+    /**
+     * Factory method for building [[DataStoreRequestBuilder]].
+     *
+     * @param serviceName The name of the service in the API.
+     * @param serviceVersion The version of the service.
+     * @param settings The [[OlpClientSettings]] instance.
+     * @param hrn A HERE Resource Name ([[HRN]]) of the catalog.
+     * @param abortSignal A signal object that allows you to communicate with a request (such as the `fetch` request)
+     * and, if required, abort it using the `AbortController` object.
+     *
+     * For more information, see the [`AbortController` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortController).
+     *
+     * @returns The Promise with constructed [[DataStoreRequestBuilder]].
+     */
+    public static async create(
```

Comment:
Is it not possible to instead of exporting the symbols from core using the RequestFactory from core here.
E.g.

```ts
public static async create(...): { core.RequestFactory.create(...); }
```

Reply:
updated, a good trick